modules/objc/examples/code_snippets/SampleCodeTest.m

```diff
@@ -205,6 +213,9 @@ - (void) dontTestDatabaseEncryption {
     // tag::database-encryption[]
     CBLDatabaseConfiguration *config = [[CBLDatabaseConfiguration alloc] init];
     config.encryptionKey = [[CBLEncryptionKey alloc] initWithPassword:@"secretpassword"];
+    // tag::database-fullsync[]
```

Comment:
You added this tag twice...

Reply:
Good spot! Was fiddling with tag behaviour and it didn't work out. Removed dead code.